src/GeoTables.jl

```diff
@@ -54,9 +54,17 @@ appropriate format based on the file extension.
 
 - `*.geojson` via GeoJSON.jl
 """
-function save(fname, geotable)
-  if endswith(fname, ".geojson")
-    GJS.write(fname, geotable)
+function save(fname, geotable; kwargs...)
+  if endswith(fname, ".shp")
+    geotable = MeshData(geotable)
+    geoms = domain(geotable)
+    if !isa(geoms[1], Multi) & !isa(geoms[1], Point)
+      geoms = GeometrySet(map(x -> Multi([x]), geoms))
+      geotable = meshdata(geoms, etable = values(geotable))
+    end
```

Comment:
Can you comment this code block? The `geotable` is a Tables.jl table with a column `:geometry`. Shapefile.jl doesn't support this format? Notice that MeshData(geotable)  shouldn't be necessary, and you can get the column of geometries with `geotable.geometry` as usual.

If all we need is convert non-Multi geometries to Multi before saving to disk, we can do so with:

```julia
newgeoms = [geom isa Multi ? geom : Multi([geom]) for geom in geotable.geometry]
newtable = georef(values(geotable), newgeomsi)
```

Reply:
We could even avoid using `georef` and just use the GeoTable type to wrap a new table with the updated geometry column.

Reply:
Yes, the main reason is that we need to convert to `Multi` with exception of `Points`. I will try your suggestion.

Reply:
Is this a limitation of the shp file format or the Shapefile.jl implementation? Should this get fixed somewhere else instead of here?

Reply:
Are you ok with something like the following?:
```julia   
newgeoms = [geom isa Multi ? geom : Multi([geom]) for geom in geotable.geometry]
meshdata(GeometrySet(newgeoms), etable = values(geotable))
```

Reply:
If we could avoid changing the geotable type before saving that would be ideal. I think the question we need to ask first is if this fix should go in Shapefile.jl instead.

Reply:
> Can you comment this code block? The `geotable` is a Tables.jl table with a column `:geometry`. Shapefile.jl doesn't support this format? Notice that MeshData(geotable) shouldn't be necessary, and you can get the column of geometries with `geotable.geometry` as usual.
> 
> If all we need is convert non-Multi geometries to Multi before saving to disk, we can do so with:
> 
> ```julia
> newgeoms = [geom isa Multi ? geom : Multi([geom]) for geom in geotable.geometry]
> newtable = georef(values(geotable), newgeomsi)
> ```

I think we should avoid redefining the data if not needed (Point and Multi), for this reason I used a conditional.

Reply:
Sorry, I missed some of your comments, it was weird that it was not showing before. I also thought if that should be a fix in `Shapefile.jl`, but I am not sure. It is not really a bug because `Shapefile.jl` read and write geometries as `Multi` with exception to `Points`, so they do not have (and do not need) a method to read or write a simple `LineString` or `Polygon`.